src/app/compare/CompareSuggestions.tsx

```diff
@@ -83,7 +87,7 @@ export default memo(function CompareSuggestions({
           onClick={() => onQueryChanged(query)}
         >
           {buttonLabel.map((l) => (typeof l === 'string' ? <span key={l}>{l}</span> : l))}
-          <span key="itemcount">({items.length})</span>
+          {!query.includes('id:') && <span>({items.length})</span>}
```

Comment:
I think the query will include id if you manually exclude any item by clicking the X on it